include/z64scene.h

```diff
@@ -417,4 +419,139 @@ typedef enum {
     /* 0x6E */ SCENE_ID_MAX
 } SceneID;
 
+
+typedef enum {
+    /* 0x00 */ SCENECMD_ID_SPAWN_LIST,
+    /* 0x01 */ SCENECMD_ID_ACTOR_LIST,
+    /* 0x02 */ SCENECMD_ID_ACTOR_CUTSCENE_CAM_LIST,
+    /* 0x03 */ SCENECMD_ID_COL_HEADER,
+    /* 0x04 */ SCENECMD_ID_ROOM_LIST,
+    /* 0x05 */ SCENECMD_ID_WIND_SETTINGS,
+    /* 0x06 */ SCENECMD_ID_ENTRANCE_LIST,
+    /* 0x07 */ SCENECMD_ID_SPECIAL_FILES,
+    /* 0x08 */ SCENECMD_ID_ROOM_BEHAVIOR,
+    /* 0x09 */ SCENECMD_ID_UNUSED_9,
+    /* 0x0A */ SCENECMD_ID_MESH,
+    /* 0x0B */ SCENECMD_ID_OBJECT_LIST,
+    /* 0x0C */ SCENECMD_ID_POS_LIGHT_LIST,
+    /* 0x0D */ SCENECMD_ID_PATH_LIST,
+    /* 0x0E */ SCENECMD_ID_TRANSI_ACTOR_LIST,
+    /* 0x0F */ SCENECMD_ID_LIGHT_LIST,
+    /* 0x10 */ SCENECMD_ID_TIME_SETTINGS,
+    /* 0x11 */ SCENECMD_ID_SKYBOX_SETTINGS,
+    /* 0x12 */ SCENECMD_ID_SKYBOX_DISABLES,
+    /* 0x13 */ SCENECMD_ID_EXIT_LIST,
+    /* 0x14 */ SCENECMD_ID_END,
+    /* 0x15 */ SCENECMD_ID_SOUND_SETTINGS,
+    /* 0x16 */ SCENECMD_ID_ECHO_SETTINGS,
+    /* 0x17 */ SCENECMD_ID_CUTSCENE_LIST,
+    /* 0x18 */ SCENECMD_ID_ALTERNATE_HEADER_LIST,
+    /* 0x19 */ SCENECMD_ID_MISC_SETTINGS,
+    /* 0x1A */ SCENECMD_ID_TEXTURE_ANIM_LIST,
+    /* 0x1B */ SCENECMD_ID_ACTOR_CUTSCENE_LIST,
+    /* 0x1C */ SCENECMD_ID_MINIMAP_INFO,
+    /* 0x1D */ SCENECMD_ID_UNUSED_1D,
+    /* 0x1E */ SCENECMD_ID_MINIMAP_COMPASS_ICON_INFO
+} SceneCommandTypeID;
+
+#define SCENECMD_SPAWN_LIST(spawnNum, spawnList)                 \
+    { SCENECMD_ID_SPAWN_LIST, spawnNum, CMD_PTR(spawnList) }
+
+#define SCENECMD_ACTOR_LIST(actorNum, actorList)                 \
+    { SCENECMD_ID_ACTOR_LIST, actorNum, CMD_PTR(actorList) }
+
+#define SCENECMD_CAM_LIST(camNum, camList)       \
+    { SCENECMD_ID_ACTOR_CUTSCENE_CAM_LIST, camNum, CMD_PTR(camList) }
+
+#define SCENECMD_COL_HEADER(colHeader)                    \
+    { SCENECMD_ID_COL_HEADER, 0, CMD_PTR(colHeader) }
+
+#define SCENECMD_ROOM_LIST(roomNum, roomList)                 \
+    { SCENECMD_ID_ROOM_LIST, roomNum, CMD_PTR(roomList) }
+
+#define SCENECMD_WIND_SETTINGS(xDir, yDir, zDir, strength)                     \
+    { SCENECMD_ID_WIND_SETTINGS, 0, CMD_BBBB(xDir, yDir, zDir, strength) }
+
+#define SCENECMD_ENTRANCE_LIST(entranceList)                    \
+    { SCENECMD_ID_ENTRANCE_LIST, 0, CMD_PTR(entranceList) }
+
+#define SCENECMD_SPECIAL_FILES(elfMessageFile, specialObject)               \
+    { SCENECMD_ID_SPECIAL_FILES, elfMessageFile, CMD_W(specialObject) }
+
```

Comment:
I dont know much about this command, but it seems odd that its expecting something called `elfMessage` specifically. Is the command for navis hints specifically, or is it intended for other files as well?
If the former then we may want a more specific name for the command. if the latter, then the args should be more general

Reply:
this is for navi hints specifically

Reply:
alright, then id say we can consider a better name than "special files"

Reply:
extra files? It's kind of weird to name because the two data points are completely irrelevant. specialObject should maybe be keepObjectId or seg5ObjectId

Reply:
ah i see, it sets both segment 5 and elf message. for some reason i thought that was done elsewhere.
Yeah naming that is hard. Extra is a little bit better, since I dont think theyre any more special than anything else.
And yea, agreed with `keepObjectId`